<launch>
    <!--Package is the name of the package as inthe package.xml is defined, type is the node name that is the name of
    of the executable created, finally name is just a given name (Just the name you will see when using "rosnode list").
    output=screen is meant to redirect the stdout or logs to the terminal where the launch was made.-->
    <node pkg="rosify_difodo" type="ros_difodo" name="rosify_difodo_node" output="screen"/>

    <param name="input_depth_topic" value="/camera/depth/image_rect_raw" type="str" />
    <param name="output_odom_topic" value="/difodo/odometry" type="str" />

    <param name="rows_orig" value="480" type="int" />
    <param name="cols_orig" value="640" type="int" />

    <param name="depth_pixel_scale" value="1000" type="int" />

    <param name="min_depth_value_filter" value="0.5" type="double" />
    <param name="max_depth_value_filter" value="4.5" type="double" />

    <param name="downsample" value="4" type="int" />

    <param name="objective_fps" value="0" type="int" />

    <param name="ctf_levels" value="5" type="int" />

    <param name="fovh_degrees" value="74.0" type="double" />
    <param name="fovv_degrees" value="62.0" type="double" />

    <param name="fast_pyramid" value="false" type="bool" />
</launch>